grpc-utils/src/main/java/com/glegoux/grpc/server/GrpcServerSimple.java

```diff
@@ -129,4 +121,9 @@ public int getMonitoringPort() {
     public int getNumberOfThreads() {
         return numberOfThreads;
     }
+
+    public List<BindableService> getServices() {
```

Comment:
this is used? I do not see where in the modifications

Reply:
I removed it